tracer/test/Datadog.Trace.Tests/TraceContextTests.cs

```diff
@@ -173,9 +173,9 @@ public void FullFlushShouldNotPropagateSamplingPriority()
         [Theory]
         [InlineData(true)]
         [InlineData(false)]
-        public void PartialFlushShouldPropagateMetadata(bool inAASContext)
+        public void ChunksSentAfterRootSpanShouldContainAASMetadataAndSamplingPriority(bool inAASContext)
         {
-            const int partialFlushThreshold = 2;
+            const int partialFlushThreshold = 3;
 
             Span CreateSpan() => new Span(new SpanContext(42, SpanIdGenerator.ThreadInstance.CreateNew()), DateTimeOffset.UtcNow);
```

Comment:
Just to make sure we don't invoke the partial flush code path, could you remove the partial flush configuration that's added in this test case?